clients/axios/axios-client-sdk.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { createUser } from "./sdk/client";
+import * as moxios from "moxios";
+import { CreateUserResponse } from "./sdk/types";
+
+const expectedResponse: CreateUserResponse = {
+  success: true,
+  created_at: "2018-01-01"
+};
+
+describe("TypeScript axios client sdk test", () => {
```

Comment:
I'd also like to see a test of a `GET` endpoint.

Reply:
sure it will be there, just get the POST first to get everyone to agreement if this is a good way to test it out